src/box/box.cc

```diff
@@ -4426,6 +4433,15 @@ box_process_vote(struct ballot *ballot)
 	vclock_copy(&ballot->vclock, &replicaset.vclock);
 	vclock_copy(&ballot->gc_vclock, &gc.vclock);
 	ballot->bootstrap_leader_uuid = bootstrap_leader_uuid;
+	if (*INSTANCE_NAME != '\0') {
```

Comment:
Can't `INSTANCE_NAME` be NULL? In some other places I see we check for it, potential null dereference maybe?

Reply:
No, it's defined as `char INSTANCE_NAME[NODE_NAME_SIZE_MAX];` in `src/box/replication.cc`.

We only have checks for `*INSTANCE_NAME != 0` in some places.

Reply:
My bad) thanks for bringing my attention to it.